README.md

```diff
@@ -327,8 +327,82 @@ Templates in React are written inside the JavaScript file using the [JSX languag
 
 # Filters
 
-> TODO
-> Angular's pipe character (|) 
+### AngularJS
+
+AngularJS provides filters to transform data for transform of data.<br />
+There are several built-in filters to use or you can make your own custom filters as well.<br />
+Filters can applied to view template using following syntax :  `{{ expression | filter }}`<br />
```

Comment:
Grammar correction: `Filters can be applied to view template using the following syntax:`